bin/kube-divergence-simulation

```diff
@@ -1,22 +1,24 @@
 #!/usr/bin/env bash
 
-# CLIENTS=(1 2 4 6 8 10)
-CLIENTS=(1)
-NODES=3
+CLIENTS=(1 2 4 1 2 4)
```

Comment:
Doing this is a problem that invalidates the results: this means that there are two nodes that are labeled for a run of client_number 4: because of the way the constraints are configured on launching containers, this **does not guarantee** that all of the containers for a single experiment are on the same node, therefore invalidating all of the results.